### PR TITLE
Change __version__ on release

### PR DESCRIPTION
**Pull request by Yang-33**
https://github.com/line/line-bot-sdk-python/issues/526

To make release easier, this pull request
1. Stop embedding version. Instead, code reads `__version__ `defined in `linebot/__about__.py`.
2. Add github acitions that commits generated code to master branch directly.
3. Replace `__verison__` with `${{ github.event.release.tag_name }}`  before uploading python package on release. (ref: [line-bot-java-sdk](https://github.com/line/line-bot-sdk-java/blob/6d97349df6e31a59d5a7931e4d97bbd589a459c3/.github/workflows/publish.yml))

If this change is merged, all we have to do can be
- creating release
- fixing unexpected error in CI

### CODE REVIEW

File: linebot/__about__.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 """Meta data of line-bot-sdk."""
 
 
-__version__ = '3.5.0'
+__version__ = "__LINE_BOT_SDK_PYTHON_VERSION__"
```
Comment (Yang-33) [inline linebot/__about__.py:18]: (check point)

File: .github/workflows/auto-testing.yml [hunk 2]
```diff
@@ -24,14 +24,18 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         if [ -f requirements-dev.txt ]; then python -m pip install -r requirements-dev.txt; fi
+    - name: Ensure generate-code.py doesn't throw any error
+      run: |
+        python generate-code.py
+    - name: Update version in linebot/__about__.py
+      run: |
+        sed -i 's/__version__ = "__LINE_BOT_SDK_PYTHON_VERSION__"/__version__ = "12.34.5"/g' linebot/__about__.py
```
Comment (tokuhirom) [inline .github/workflows/auto-testing.yml:32]: What's `12.34.5`?

Comment (Yang-33) [reply .github/workflows/auto-testing.yml:32]: I should have written why this uses `12.34.5`...
Tox (test library) rejects running test if version is illegal. As a workaround, this embeds dummy version.

**Review by tokuhirom — APPROVED** [review]
LGTM

**Pull request merged**